index.js

```diff
@@ -187,7 +187,8 @@ async function listAll(owner, title, author, { ignoreChecks = false } = {}) {
       continue;
     }
 
-    if (!pr.title.endsWith(title)) {
+    const titleRegex = new RegExp(`${title}$`, "i");
+    if (!titleRegex.test(pr.title)) {
```

Comment:
If `title` includes some regex special characters (e.g. `.`) then this could behave unexpectedly. We could mess around with Regex escaping, but I reckon something like this would be simpler:

```suggestion
if (!pr.title.toLowerCase().endsWith(title.toLowerCase())) {

Reply:
Ah, much better! Thank you!